ocp_resources/virtual_machine_cluster_preferences.py

```diff
@@ -1,5 +1,6 @@
-from ocp_resources.resource import NamespacedResource
+from ocp_resources.resource import Resource
 
 
-class VirtualMachineClusterPreference(NamespacedResource):
-    api_group = NamespacedResource.ApiGroup.INSTANCE_TYPE_KUBEVIRT_IO
+class VirtualMachineClusterPreference(Resource):
+    api_group = Resource.ApiGroup.INSTANCE_TYPE_KUBEVIRT_IO
+    api_version = Resource.ApiVersion.V1ALPHA2
```

Comment:
api_version is not needed. we take it automaticlly.

Reply:
Are you sure it's not needed? the test passes only when I add it...